src/Parsers/ParserDropQuery.cpp

```diff
@@ -68,14 +70,24 @@ bool parseDropQuery(IParser::Pos & pos, ASTPtr & node, Expected & expected, cons
         if (s_if_empty.ignore(pos, expected))
             if_empty = true;
 
-        if (!name_p.parse(pos, table, expected))
-            return false;
-
-        if (s_dot.ignore(pos, expected))
+        while (true)
```

Comment:
I think we have a class called `ParserNameList` using to parse list of table names. you don't have to do it from scratch.

Reply:
Thanks, yes, changed.